tox.ini

```diff
@@ -83,6 +83,7 @@ deps =
     coverage[toml]
     responses
     fs
+    hypothesis
```

Comment:
Are we actually using this? IMO it would be better to remove everywhere it is used.

Reply:
@sed-i since you add it, it's possible to remove this?

Reply:
Seems to only be used by:

```
class TestPebblePlan(unittest.TestCase):
    """Feature: Multi-unit loki deployments.
    Background: TODO
    """
```
What does this test do? Who knows? It appears to be nothing and 100% overlap with other tests.

Multi-unit testing can be (and should be, and is) done in integration tests anyway. Just delete this entire testcase, and no more hypothesis anyway.